app/subscriber/src/features/my-searches/MySearches.tsx

```diff
@@ -66,63 +65,72 @@ export const MySearches = () => {
 
   return (
     <styled.MySearches>
-      <SubscriberTableContainer>
-        <Row className="header">
-          <span className="label">Search Name</span>
-        </Row>
-        {sortedMyFilters.map((filter, index) => {
-          const keywords = filter.settings?.search ? filter.settings.search : '';
+      <Grid
```

Comment:
Replaced with grid for simplicity and consistency.  It's not perfect, but it's much better than before.